src/main/scala/collections/CollectionsMain.scala

```diff
@@ -0,0 +1,41 @@
+package collections
+
+import scala.io.Source
+
+object CollectionsMain extends App {
+  val lines = Source.fromResource("atussum.csv").getLines.take(10000)
+  val headers = lines.next.split(",").map(_.trim).toList
+  val rows = lines.toArray.map(_.split(",").map(_.trim)).map(_.zipWithIndex)
+
+  val primaryNeedsIndices = findIndices(List("t01", "t03", "t11", "t1801", "t1803"))
```

Comment:
It would be much more readable if You have used some model to describe each row (and it's properties we are interested about). It should also be as easy to read and change, as possible. For example, in several weeks we could potentially get some new data, indicated with different t** values => it should be easy to make such changes. It would be also much easier to test that way.

Reply:
added Row case class

Reply:
well, I meant each row may be represented with a case class object (f.e. Respondent), that incapsulates every criteria we are interested in, like sex, age etc. and is produced as a result of parsing a row. And then we can manipulate not abstract mapping, but properties with meaningful names

Reply:
added case class Respondent with the proper fields to represent rows